pkg/chartutil/save.go

```diff
@@ -56,6 +57,14 @@ func SaveDir(c *chart.Chart, dest string) error {
 		}
 	}
 
+	// Save values.schema.json if it exists
+	if c.Schema != nil {
```

Comment:
Do we want to check if the schema is valid here as well?

```
if !json.Valid(c.Schema) {
	return errors.New("Invalid JSON in " + SchemafileName)
}
```

Reply:
I don't know. But on [line 54](https://github.com/helm/helm/blob/3c977a905518ad54f92df823ed5b4ca488fd6aeb/pkg/chartutil/save.go#L54), `values.yaml` is marshaled without any error being considered. Compare this to [line 154](https://github.com/helm/helm/blob/3c977a905518ad54f92df823ed5b4ca488fd6aeb/pkg/chartutil/save.go#L154), where any marshaling error is caught and returned. I think that's why I did the validation in `writeTarContents`, but not in `SaveDir`.